### DISCUSSION TITLE
RT

### DISCUSSION THREAD

**Discussion by GB12139** [community/community#70999] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![屏幕截图(4)](https://github.com/community/community/assets/145318049/47f247ed-828c-4843-abe4-f60cfb7a3bde)
![屏幕截图(4)](https://github.com/community/community/assets/145318049/091b682f-3c9d-4288-9d04-662bcc8e9587)

**Comment by queenofcorgis** [comment]
Hi @GB12139 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
